airflow/providers/docker/decorators/docker.py

```diff
@@ -128,9 +190,15 @@ def execute(self, context: Context):
 
     @property
     def pickling_library(self):
```

Comment:
typing please

Reply:
Typing of lazy loaded module? 
Do you have a suggestion how it would looks like?

Reply:
[`ModuleType`](https://docs.python.org/3/library/types.html#types.ModuleType) exists for build module dynamically, it not make a sense to use it for type annotations. 

What the point here to have it rather than `Any` / `object` here? What benefits it would granted?

Reply:
There is no problem to add it, I just need to know reason why we should have it on this property